Block/NodeType/CategoryChild.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+namespace Snowdog\Menu\Block\NodeType;
+
+use Magento\Framework\View\Element\Template\Context;
+use Magento\Framework\Registry;
+use Magento\Catalog\Api\CategoryRepositoryInterface;
+use Snowdog\Menu\Model\TemplateResolver;
+use Snowdog\Menu\Model\NodeType\Category as ModelCategory;
+
+class CategoryChild extends Category
+{
+    /**
+     * @var string
+     */
+    protected $defaultTemplate = 'menu/node_type/category_child.phtml';
+    /**
+     * @var string
+     */
+    protected $nodeType = 'category_child';
+    protected $categoryRepository;
```

Comment:
Please add the annotation for this property:

```php
/**
 * @var CategoryRepositoryInterface
 */
protected $categoryRepository;
```

Reply:
done